app/models/customer.py

```diff
@@ -0,0 +1,32 @@
+from flask import current_app
+from sqlalchemy.orm import backref
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    videos = db.relationship("Rental", backref="customer", lazy=True)
+    
+    def to_json(self): 
+        to_json = {
+                "id": self.customer_id,
+                "name": self.name,
+                "registered_at": self.registered_at,
+                "postal_code": self.postal_code, 
+                "phone": self.phone,
+                "videos_checked_out_count": self.videos_checked_out_count              
```

Comment:
Once you have the relationship the `videos_checked_out_count` is just the length of the `rentals` that haven't been checked in:

```suggestion
            "videos_checked_out_count": len([rental for rental in self.rentals if not rental.check_in_date])
```